src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs

```diff
@@ -637,6 +637,82 @@ public static void SetMode360Capability(MediaControlCapabilitySupport support)
                 ThrowIfError("Failed to set 360 mode capability.");
         }
 
+        /// <summary>
+        /// Sets the <see cref="MediaControlCapabilitySupport"/> indicating playback position is supported or not.
+        /// </summary>
+        /// <param name="support">A value indicating whether the playback position is supported or not.</param>
+        /// <exception cref="InvalidOperationException">
+        ///     The server is not running .<br/>
+        ///     -or-<br/>
+        ///     An internal error occurs.
+        /// </exception>
+        /// <exception cref="ArgumentException"><paramref name="support"/> is invalid.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public static void SetPlaybackPositionCapability(MediaControlCapabilitySupport support)
+        {
+            ValidationUtil.ValidateEnum(typeof(MediaControlCapabilitySupport), support, nameof(support));
+
+            Native.SetSimpleCapability(Handle, MediaControlNativeCapabilityCategory.PlaybackPosition, support).
+                ThrowIfError("Failed to set playback position capability.");
+        }
+
+        /// <summary>
+        /// Sets the <see cref="MediaControlCapabilitySupport"/> indicating playlist is supported or not.
+        /// </summary>
+        /// <param name="support">A value indicating whether the playlist is supported or not.</param>
+        /// <exception cref="InvalidOperationException">
+        ///     The server is not running .<br/>
+        ///     -or-<br/>
+        ///     An internal error occurs.
+        /// </exception>
+        /// <exception cref="ArgumentException"><paramref name="support"/> is invalid.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public static void SetPlaylistCapability(MediaControlCapabilitySupport support)
+        {
+            ValidationUtil.ValidateEnum(typeof(MediaControlCapabilitySupport), support, nameof(support));
+
+            Native.SetSimpleCapability(Handle, MediaControlNativeCapabilityCategory.Playlist, support).
+                ThrowIfError("Failed to set playlist capability.");
+        }
+
+        /// <summary>
+        /// Sets the <see cref="MediaControlCapabilitySupport"/> indicating client custom is supported or not.
```

Comment:
What is the "client custom" capability? Will the API user understand that?

Reply:
I changed it to `CustomCommand`.